_src/pldoc/static/sass/_components.scss

```diff
@@ -35,42 +35,46 @@
 // ------------------------------
 // #HEADER
 // ------------------------------
-.pldoc-header-site-title {
-    display: inline-block;
-    vertical-align: middle;
+.pldoc-header-site {
 
-    @include susy-breakpoint($bp-large, $susy) {
-        margin: 0;
-    }
+    .pldoc-header-site-title {
+        display: inline-block;
+        vertical-align: middle;
 
-    .title-name, .title-description {
-        display: block;
-    }
+        @include susy-breakpoint($bp-large, $susy) {
+            margin: 0;
+        }
 
-    .title-name {
-        margin-bottom: spacing-vertical(x-small);
-        color: palette(grayscale, white);
-        font-size: font-size(base);
-        font-weight: font-weight(bold);
-        text-transform: uppercase;
-        letter-spacing: rem(1);
+        .title-name {
+            display: block;
+        }
 
-        // STATE: hover, active, focus
```

Comment:
If its a rebase against the master the diff wouldn't show up. Can you please revert this file/section based on what master has if you don't have any deliberate changes?